setuptools_rust/extension.py

```diff
@@ -47,6 +47,10 @@ class RustExtension:
       optional : bool
         if it is true, a build failure in the extension will not abort the
         build process, but instead simply not install the failing extension.
+      py_limited_api : bool
+        Same as `py_limited_api` on `setuptools.Extension`. Note that if you
+        set this to True, your extension must pass the appropriate feature
+        flags to pyo3 (ensuring that `abi3` feature is enabled).
```

Comment:
I'd like to wait to merge because there was talk of renaming the `abi3` feature in pyo3. (Usually features should add to the lib, rather than `abi3` which currently takes things away.)

Reply:
Sure, that's reasonable.